src/app/pages/wallets/my-wallets/pages-wallets-my-wallets.service.ts

```diff
@@ -28,4 +29,8 @@ export class PagesWalletsMyWalletsService {
         map(response => new MyWallet(response)),
     );
   }
+
+  public deleteWallet(wallet: MyWallet): Observable<void> {
```

Comment:
This method also should be tested, you can do this based on already existing tests